src/TLuaInterpreter.h

```diff
@@ -399,6 +396,14 @@ class TLuaInterpreter : public QThread
     static int getProfileName( lua_State * );
     static int raiseGlobalEvent( lua_State * );
 
+public slots:
+    void slot_replyFinished( QNetworkReply * );
+    void slotPurge();
+    void slotDeleteSender();
+
+private:
+    TGatekeeperThread* mpGatekeeperThread;
+    QNetworkAccessManager* mpFileDownloader;
```

Comment:
Might explain why we get up to 30 crashes in the `QSharedNetworkSessionManager` destructor a month on one of our _least used_ platforms, nevermind Windows.